jadx-gui/src/main/resources/i18n/Messages_de_DE.properties

```diff
@@ -229,7 +229,7 @@ certificate.serialSHA1=SHA-1 Fingerabdruck
 certificate.serialSHA256=SHA-256 Fingerabdruck
 certificate.serialPubKeyY=Y
 
-apkSignature.signer=Unterzeichner
+apkSignature.signer=Signieren
```

Comment:
In the context it used we need substantive, not a verb. It is used for displaying the signatures of the APK, therefore "Unterzeichner" ist the better translation.